refactoring: Migrate to netstandard and update to cake 0.28.0

nalla opened the pull request.

MilovanovM commented:
that's great, thanks for the PR! Have you had a chance to review my netstandard2 branch? https://github.com/MilovanovM/cake-yarn/tree/netstandard2. Have I missed something there and should deprecate it?

nalla commented:
Well I changed the build script and changed version to be calculated from gitversion tool - was easier to just start from current master again.

FYI: With current gitversion settings, all builds from master will be an RC and all tagged commits will be a final release.
